  openpose_python.cpp
Z:\dcongtinh On My Mac\openpose\include\openpose/flags.hpp(49,1): warning C4245: 'initializing': conversion from 'int' to 'google::uint64', signed/unsigned mismatch
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\include\xmemory(671,82): warning C4244: 'initializing': conversion from '_Ty' to '_Objty', possible loss of data
          with
          [
              _Ty=Py_ssize_t
          ]
          and
          [
              _Objty=int
          ]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\include\xmemory(1498): message : see reference to function template instantiation 'void std::_Default_allocator_traits<_Alloc>::construct<_Ty,__int64&>(_Alloc &,_Objty *const ,__int64 &)' being compiled
          with
          [
              _Alloc=std::allocator<int>,
              _Ty=int,
              _Objty=int
          ]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\include\xmemory(1498): message : see reference to function template instantiation 'void std::_Default_allocator_traits<_Alloc>::construct<_Ty,__int64&>(_Alloc &,_Objty *const ,__int64 &)' being compiled
          with
          [
              _Alloc=std::allocator<int>,
              _Ty=int,
              _Objty=int
          ]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\include\xmemory(1533): message : see reference to function template instantiation 'void std::_Uninitialized_backout_al<_Alloc>::_Emplace_back<_Ty&>(_Ty &)' being compiled
          with
          [
              _Alloc=std::allocator<int>,
              _Ty=Py_ssize_t
          ]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\include\xmemory(1533): message : see reference to function template instantiation 'void std::_Uninitialized_backout_al<_Alloc>::_Emplace_back<_Ty&>(_Ty &)' being compiled
          with
          [
              _Alloc=std::allocator<int>,
              _Ty=Py_ssize_t
          ]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\include\vector(1591): message : see reference to function template instantiation 'int *std::_Uninitialized_copy<_Iter,std::allocator<int>>(const _InIt,const _InIt,int *,_Alloc &)' being compiled
          with
          [
              _Iter=__int64 *,
              _InIt=__int64 *,
              _Alloc=std::allocator<int>
          ]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\include\vector(497): message : see reference to function template instantiation 'int *std::vector<int,std::allocator<int>>::_Ucopy<_Iter>(_Iter,_Iter,int *)' being compiled
          with
          [
              _Iter=__int64 *
          ]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\include\vector(497): message : see reference to function template instantiation 'int *std::vector<int,std::allocator<int>>::_Ucopy<_Iter>(_Iter,_Iter,int *)' being compiled
          with
          [
              _Iter=__int64 *
          ]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\include\vector(508): message : see reference to function template instantiation 'void std::vector<int,std::allocator<int>>::_Range_construct_or_tidy<__int64*>(_Iter,_Iter,std::forward_iterator_tag)' being compiled
          with
          [
              _Iter=__int64 *
          ]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\include\vector(508): message : see reference to function template instantiation 'void std::vector<int,std::allocator<int>>::_Range_construct_or_tidy<__int64*>(_Iter,_Iter,std::forward_iterator_tag)' being compiled
          with
          [
              _Iter=__int64 *
          ]
Z:\dcongtinh On My Mac\openpose\python\openpose\openpose_python.cpp(413): message : see reference to function template instantiation 'std::vector<int,std::allocator<int>>::vector<std::_Vector_iterator<std::_Vector_val<std::_Simple_types<_Ty>>>,void>(_Iter,_Iter,const _Alloc &)' being compiled
          with
          [
              _Ty=pybind11::ssize_t,
              _Iter=std::_Vector_iterator<std::_Vector_val<std::_Simple_types<pybind11::ssize_t>>>,
              _Alloc=std::allocator<int>
          ]
Z:\dcongtinh On My Mac\openpose\python\openpose\openpose_python.cpp(413): message : see reference to function template instantiation 'std::vector<int,std::allocator<int>>::vector<std::_Vector_iterator<std::_Vector_val<std::_Simple_types<_Ty>>>,void>(_Iter,_Iter,const _Alloc &)' being compiled
          with
          [
              _Ty=pybind11::ssize_t,
              _Iter=std::_Vector_iterator<std::_Vector_val<std::_Simple_types<pybind11::ssize_t>>>,
              _Alloc=std::allocator<int>
          ]
Z:\dcongtinh On My Mac\openpose\python\openpose\openpose_python.cpp(397): message : see reference to class template instantiation 'pybind11::detail::descr<13>' being compiled
Z:\dcongtinh On My Mac\openpose\3rdparty\pybind11\include\pybind11\cast.h(1556): message : see reference to class template instantiation 'pybind11::detail::descr<8>' being compiled
Z:\dcongtinh On My Mac\openpose\3rdparty\pybind11\include\pybind11\cast.h(1554): message : see reference to class template instantiation 'pybind11::detail::descr<5>' being compiled
Z:\dcongtinh On My Mac\openpose\3rdparty\pybind11\include\pybind11\cast.h(1109): message : see reference to class template instantiation 'pybind11::detail::descr<7>' being compiled
     Creating library Z:/dcongtinh On My Mac/openpose/python/openpose/Release/pyopenpose.lib and object Z:/dcongtinh On My Mac/openpose/python/openpose/Release/pyopenpose.exp
  Generating code
  Finished generating code
  pyopenpose.vcxproj -> Z:\dcongtinh On My Mac\openpose\python\openpose\Release\pyopenpose.cp36-win_amd64.pyd
